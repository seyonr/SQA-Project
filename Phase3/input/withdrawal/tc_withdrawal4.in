login 
standard
54321
Jane Doe
W
800