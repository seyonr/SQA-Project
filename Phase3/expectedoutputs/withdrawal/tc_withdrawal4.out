Welcome to the banking system!
Do you want to login or exit : 
Enter login type (admin or standard): 
Enter account number: 
Enter account name: 
Login successful!

What do you want to do today?
W – Withdrawal
T - Transfer
P - Paybill
D - Deposit
C - Create
DE - Delete
DI - Disable
CH - Change Plan
L – Logout
How much would you like to withdraw : 
Standard users can’t withdraw above 500, session terminated!
